app/views/admin/users/_user.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="col-3">
+  <%= user.email %>
+</div>
+<div class="col-2">
+  <%= user.locked_at? ? 'blocked' : 'active' %>
+</div>
+<div class="col-3">
+  <%= form_for([:admin,user]) do |f| %>
+    <%= f.select(:role, User.roles.keys.map {|role| [role.titleize,role]}) %>
+    <%= f.submit 'Change Role', :class => 'button-xs' %>
+  <% end %>
+</div>
+<div class="col-2">
+  <%= link_to('Lock', admin_lock_access_path(user), :data => { :confirm => 'Are you sure?' }, :method => :post, :class => 'button-xs') unless user == current_user %>
```

Comment:
it would be better to have something like:
```ruby
<%= link_to 'Lock', admin_lock_access_path(user),
            data: { confirm: 'Are you sure?' }, 
           method: :put, 
           class: 'button-xs' %>
```
- avoid using of old hash style when it's unnecessary
- `put` or `patch` - is better to use for updates, while `post` for create

Reply:
resolved with patch method